esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -737,18 +764,23 @@ void WsWuInfo::doGetGraphs(IArrayOf<IEspECLGraph>& graphs)
 
         if (version >= 1.53)
```

Comment:
@ghalliday Does this version number need to be incremented?

Reply:
No because the specification of the service interface hasn't changed - only the implementation.